Formula/djl-bench.rb

```diff
@@ -0,0 +1,30 @@
+class DjlBench < Formula
+  desc "Command-line tool that helps to benchmark the model on different platforms"
+  homepage "https://github.com/deepjavalibrary/djl/tree/master/extensions/benchmark"
+  url "https://publish.djl.ai/djl-bench/0.12.0/benchmark-0.12.0.tar"
+  sha256 "778df005951c546c4e8e4e22dd62e1bc64b413334e0951ffbd8d06be5cc6ce19"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+    mv "bin/benchmark", "bin/djl-bench"
+    libexec.install Dir["*"]
+    env = { APP_HOME: "${APP_HOME:-#{var}}" }
+    env.merge!(Language::Java.overridable_java_home_env)
+    (bin/"djl-bench").write_env_script "#{libexec}/bin/djl-bench", env
+  end
+
+  service do
+    run [opt_bin/"djl-bench", "run"]
+    keep_alive true
+  end
+
+  test do
+    ENV["APP_HOME"] = testpath
+    cp_r Dir["#{libexec}/*"], testpath
+    output = shell_output("#{bin}/djl-bench -h")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Hmm, so it seems like I'm having trouble downloading the MXNet engine on the ARM based machine. 

The previous `djl-bench -h` command works with djl-bench when installed properly. Should I revert to that ?

Reply:
Does `djl-bench` download `libmxnet.dylib` at runtime? We don't want that. We should build that from source instead. Also, isn't the test failure a sign that `djl-bench` isn't working?

Reply:
Yes, the djl-bench downloads and uses `libmxnet.dylib` when it runs. Since, it's not able to download/use it here, it's not able to run. Its designed to download (and eventually cache) these dependencies at the first time of its run.

Reply:
We should build `libmxnet.dylib` during `install` instead, so that it's not downloaded at runtime. We've removed formulae that do similar things from Homebrew/core before. Building `libmxnet.dylib` should also hopefully allow this to work on ARM.